data/reporting-guide.md

```diff
@@ -0,0 +1,114 @@
+title: Reporting Guide
+header: Reporting Guide
+bloglike: true
+nav: reporting-guide
+---
+
+At PyCon India 2023, we are committed to providing a safe and welcoming
+environment for all attendees. We take incidents of harassment, misconduct, or
+violations of our [Code Of Conduct](../code-of-conduct) very seriously. This
+Reporting Guide aims to provide clear instructions on how to report any concerns
+and the steps we will take to address reported incidents.
+
+## Reporting Process
+
+If you experience or witness any behavior that you believe violates the PyCon
+India Code Of Conduct, we encourage you to report it as soon as possible.
+Reports can be made through the following channels:
+
+a. **Contact a Volunteer:** Look for individuals wearing "PyCon India Volunteer"
+badges and approach them to make your report in person.
+
+b. **Email:** You can send an email to
+[coc@in.pycon.org](mailto:coc@in.pycon.org) to submit your report
+electronically.
+
+c. **In-person:** If you prefer to discuss the matter in person, please notify
+any member of the PyCon India organizing team or the Code Of Conduct Committee.
+
+## Information to Include in Your Report
+
+When filing a report, please provide as much relevant information as possible.
+The following details are essential to help us understand and address the
+situation:
+
+- **Your contact information:** So we can get in touch with you if additional
+  information is needed during the investigation.
+
+- **Names and details of involved parties:** Include the names (real, nicknames,
+  or pseudonyms) of all individuals involved in the incident, including any
+  witnesses.
+
+- **Date, time, and location of the incident:** Be as specific as possible about
+  when and where the incident occurred.
+
+- **Description of the incident:** Provide a detailed account of what happened,
+  including any offensive language or actions involved.
+
+- **Evidence, if available:** If there is any publicly available evidence (e.g.,
+  screenshots, recordings, links to relevant discussions), please include them
+  in your report.
+
+- **Additional context:** Share any context you believe might be relevant to the
+  incident.
+
+- **Is the incident ongoing?:** Let us know if the incident is ongoing or if
+  there is an immediate threat to someone's safety.
+
+## Confidentiality
+
+All reports will be treated with strict confidentiality. We understand the
+sensitive nature of such matters and will protect the privacy of all parties
+involved to the best of our ability. In some cases, we may need to make a public
+statement, but the identities of victims and reporters will remain confidential
+unless instructed otherwise.
+
+## Reporting Concerns about Volunteers or Organizers
+
+If your report concerns a possible violation by a volunteer or organizer, you
+can make your report directly to the Chair of the Code Of Conduct Committee,
+whose contact information will be made available during the conference.
```

Comment:
Please add the contact information for Nivesh Krishna here
vniveshkrishna.98 at gmail dot com